src/Http/Request/Request.php

```diff
@@ -53,15 +53,10 @@ public function getIncludedRelationships()
         $include = $this->getQueryParam('include', []);
         $included = new Included();
 
-        if (is_string($include)) {
-            if (!strlen($include)) {
-                $included->setHidden();
-            } else {
-                $includeNames = \explode(',', $include);
-                foreach ($includeNames as $relationship) {
-                    $included->add($relationship);
-                }
-            }
+        if (is_string($include) && strlen($include)) {
```

Comment:
`stlen($include)` is there to make sure empty strings don't make it through, that will cause an exception